src/lib/utils/hooks/useGetInfoFromIds.ts

```diff
@@ -53,27 +57,27 @@ const getEntityHeaderItems = async (
   return [...newData.results, ...notFoundPlaceholders]
 }
 
-const getUserProfileItems = async (
+const getUserGroupHeaderItems = async (
   lookupList: string[],
-): Promise<UserProfile[]> => {
-  const newData = await getUserProfileWithProfilePicAttached(lookupList)
+): Promise<UserGroupHeader[]> => {
+  const newData = (await getGroupHeadersBatch(lookupList)).children
   const notFound = lookupList.filter(
-    item => newData.list.map(item => item.ownerId).indexOf(item) === -1,
+    item => newData.map(item => item.ownerId).indexOf(item) === -1,
   )
   const notFoundPlaceholders = notFound.map(item => ({
-    ...UserProfileTemplate,
+    ...UserGroupHeaderTemplate,
     ownerId: item,
     name: `Unknown User (${item})`,
   }))
 
-  return [...newData.list, ...notFoundPlaceholders]
+  return [...newData, ...notFoundPlaceholders]
 }
 
 // React hook to get user profiles or entities
 //******************************************************************************************* */
-export default function useGetInfoFromIds<T extends EntityHeader | UserProfile>(
-  props: UseGetInfoFromIdsProps,
-) {
+export default function useGetInfoFromIds<
+  T extends EntityHeader | UserGroupHeader,
+>(props: UseGetInfoFromIdsProps): T[] {
```

Comment:
Hook that stores/caches EntityHeaders or UserProfiles. Change to use UserGroupHeaders instead of UserProfiles, so it works for teams (and can be batched instead of making _n_ requests!)